toffy/fov_watcher_test.py

```diff
@@ -40,13 +42,16 @@ def _slow_copy_sample_tissue_data(dest: str, delta: int = 10):
 }
 
 
-@parametrize_with_cases('per_fov_partial, per_run', cases=WatcherCases)
-def test_watcher(per_fov_partial, per_run):
+# TODO: add tests for per_run when per_run callbacks are created
+@parametrize_with_cases('per_fov_partial, per_run, validators', cases=WatcherCases)
+def test_watcher(per_fov_partial, per_run, validators):
     with tempfile.TemporaryDirectory() as tmpdir:
 
+        RUN_DIR_NAME = 'run_XXX'
```

Comment:
Is there a reason this is a constant? If so, can it be put outside the test function?

Reply:
That's a constant because I retyped 'run_XX' into one of the functions and got confused why everything was breaking for a solid 5 minutes lol.  It should probably stay a constant but I agree, it should be in global scope.